packages/legacy/core/App/screens/Splash.tsx

```diff
@@ -129,105 +110,74 @@ const Splash: React.FC = () => {
     const initOnboarding = async (): Promise<void> => {
       try {
         // load authentication attempts from storage
-        const attemptData = await loadAuthAttempts()
-
-        const preferencesData = await AsyncStorage.getItem(LocalStorageKeys.Preferences)
-        if (preferencesData) {
-          const dataAsJSON = JSON.parse(preferencesData) as PreferencesState
-
-          dispatch({
-            type: DispatchAction.PREFERENCES_UPDATED,
-            payload: [dataAsJSON],
-          })
-        }
-
-        const migrationData = await AsyncStorage.getItem(LocalStorageKeys.Migration)
-        if (migrationData) {
-          const dataAsJSON = JSON.parse(migrationData) as MigrationState
-
-          dispatch({
-            type: DispatchAction.MIGRATION_UPDATED,
-            payload: [dataAsJSON],
-          })
-        }
-
-        const toursData = await AsyncStorage.getItem(LocalStorageKeys.Tours)
-        if (toursData) {
-          const dataAsJSON = JSON.parse(toursData) as ToursState
-
-          dispatch({
-            type: DispatchAction.TOUR_DATA_UPDATED,
-            payload: [dataAsJSON],
-          })
+        if (!store.stateLoaded) {
+          return
         }
 
-        const data = await AsyncStorage.getItem(LocalStorageKeys.Onboarding)
-        if (data) {
-          const onboardingState = JSON.parse(data) as StoreOnboardingState
-          dispatch({ type: DispatchAction.ONBOARDING_UPDATED, payload: [onboardingState] })
-          if (onboardingComplete(onboardingState, { termsVersion: TermsVersion })) {
-            // if they previously completed onboarding before wallet naming was enabled, mark complete
-            if (!store.onboarding.didNameWallet) {
-              dispatch({ type: DispatchAction.DID_NAME_WALLET, payload: [true] })
-            }
+        if (onboardingComplete(store.onboarding, { termsVersion: TermsVersion })) {
+          // if they previously completed onboarding before wallet naming was enabled, mark complete
+          if (!store.onboarding.didNameWallet) {
+            dispatch({ type: DispatchAction.DID_NAME_WALLET, payload: [true] })
+          }
 
-            // if they previously completed onboarding before preface was enabled, mark seen
-            if (!store.onboarding.didSeePreface) {
-              dispatch({ type: DispatchAction.DID_SEE_PREFACE })
-            }
+          // if they previously completed onboarding before preface was enabled, mark seen
+          if (!store.onboarding.didSeePreface) {
+            dispatch({ type: DispatchAction.DID_SEE_PREFACE })
+          }
 
-            if (!attemptData?.lockoutDate) {
+          if (!store.loginAttempt.lockoutDate) {
+            setTimeout(() => {
```

Comment:
What is this set timeout for?

Reply:
old artifact, I just realized it was still there and removed it